roles/identity-management/manage-rh-sso-identities/tasks/create_users.yml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Just a nitpick - this file is handling only **one** user, but the filename is users - with a `s` - leading a consumer to think it handles the list of users. I'd suggest renaming it to better reflect what it does (in case someone decides to source just this one file). 

Same goes for the delete file below.

Reply:
Fixed.